src/index.html

```diff
@@ -7,8 +7,85 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <div>
+        <a class="logo" href="#home">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__items">
+            <a class="is-active"
+              rel="liner"
+              href="#apples"
```

Comment:
just add two spaces to get all attributes in a column:
```
<a class="is-active"
   rel="liner"
   href="#apples"
```

Reply:
all is ready, please check